styles.css

```diff
@@ -0,0 +1,16 @@
+/* opd 1 */
+#currentStock{
+    color: rgb(255, 0, 0);
+    font-weight: bold;
+}
+
+/* opd 3 */
+#profitGoal{
+    color: rgba(95, 159, 243);
```

Comment:
Bij rgba verwacht hij nog een Alpha (opacity)

Reply:
same here, editor (visual studio code) maakt zo de RGBA wat idd wel een beetje apart is. Ik zal het onthouden om de alpha toe te voegen!